tests/decode.cpp

```diff
@@ -103,6 +103,13 @@ int main(int argc, char** argv)
     }
 
     status = decoder->start(&configBuffer);
+
+    formatInfo = decoder->getFormatInfo();
+    if (formatInfo->width && formatInfo->height) {
+        if (!output->setVideoSize(formatInfo->width, formatInfo->height))
```

Comment:
@xuguangxin the width & height set to video size is aligned. if put setVideoSize to previous if, we should do the align for the width and height.